mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -127,6 +127,8 @@ CLASS net/minecraft/class_1745 net/minecraft/entity/Entity
 	METHOD method_6923 setBounds (FF)V
 		ARG 1 width
 		ARG 2 height
+	METHOD method_6924 updateVelocity (FFF)V
+		ARG 1 speed
```

Comment:
this doesn't seem righ
![image](https://user-images.githubusercontent.com/62552372/124357356-aa65a700-dc38-11eb-922a-462858be321f.png)
best to leave it unmapped for now

Reply:
Why not? 1.14.4 has it similar:
![updateVelocity-1.14.4](https://user-images.githubusercontent.com/69988482/124359795-cf511e80-dc26-11eb-81b8-47b645f04e15.png)

Reply:
ah

Reply:
probably remove the parameter name tho since there're two of them with deceptively similar usages

Reply:
done